scripts/compare-ci-runs/compare-runs.py

```diff
@@ -0,0 +1,241 @@
+#! /usr/bin/env python3
+
+import argparse
+import shutil
+import subprocess
+import os
+import pandas as pd
+from pathlib import Path
+
+
+def get_config(ident: str) -> str:
+    """Retrieve configuration name"""
+    if ":" in ident:
+        return ident.split(":")[0]
+
+    return ident
+
+
+def download(ident: str) -> bool:
+    """Download artifacts for given configuration and CI run ID from Github"""
+    if not shutil.which("gh"):
+        print("Could not find 'gh' executable on the '$PATH'")
+        return False
+
+    if ":" not in ident:
+        print("Invalid format, expecting 'name:Github CI Run ID' for download")
+        return False
+
+    name, run = ident.split(":", 1)
+
+    ret = subprocess.run(["gh", "run", "download", "-R", "intel/intel-xpu-backend-for-triton", "-D", name, run],
+                         capture_output=True)
+
+    if ret.returncode != 0:
+        print("Downloading run artifacts with 'gh' CLI failed")
+        if ret.stdout:
+            print("Command stdout:")
+            print(ret.stdout.decode("UTF-8"))
+        if ret.stderr:
+            print("Command stderr:")
+            print(ret.stderr.decode("UTF-8"))
+        return False
+
+    return True
+
+
+def get_raw_data(args: argparse.Namespace) -> tuple[Path, Path]:
+    """Discover or download the raw data for both configurations."""
+    numDir = Path(os.getcwd()).joinpath(get_config(args.numerator))
+    denomDir = Path(os.getcwd()).joinpath(get_config(args.denominator))
```

Comment:
I think `Path()` creates a path relative to the current directory, `os.getcwd()` is not required.
```suggestion
    numDir = Path(get_config(args.numerator))
    denomDir = Path(get_config(args.denominator)).
```